packages/hadron-express/README.md

```diff
@@ -84,6 +84,22 @@ GET request with path: `http://localhost/Hello/World` will result with following
 "Hello World"
 ```
 
+### Locals (available from 2.0.0)
+
+As a third parameter, hadron delivers `locals` from your response. You can inject it's content in middlewares, f.e.
```

Comment:
a typo - `it's` should be `its`

`f.e.` - I don't think such an abbreviation exists, it should be `e.g.`